src/content/en/tools/workbox/guides/common-recipes.md

```diff
@@ -12,22 +12,43 @@ This page contains a set of example caching strategies you can use with Workbox.
 
 ## Google Fonts
 
-You can use a cache first strategy to cache the Google Fonts in your page.
-Here we've limited the cache to 30 entries to ensure we don't balloon a users
-device.
+The Google Fonts service consist of two parts:
+
+- The stylesheet with the `@font-face` definitions, which link to the font
+  files.
+- The static, revisioned font files.
+
+The stylesheet can change frequently, so it's best to use a caching strategy
+like stale while revalidate that checks for updates with every request. The font
+files themselves, on the other hand, do not change and can leverage a cache
+first strategy.
+
+Here we've limited the age of the cached font files to one year (matching the
+HTTP `Cache-Control` header) and the max entries to 30 (to ensure we don't use
+up too much storage on the user's device).
 
 ```javascript
+// Cache the Google Fonts stylesheets with a stale while revalidate strategy.
 workbox.routing.registerRoute(
-  new RegExp('https://fonts.(?:googleapis|gstatic).com/(.*)'),
+  /^https:\/\/fonts\.googleapis\.com/,
+  workbox.strategies.staleWhileRevalidate({
```

Comment:
I'm not 100% sure what the behavior would be if you use the same `google-fonts` cache name for two different strategies, and one of those strategies uses cache expiration.

I *think* what would end up happening is that only the entries that were created by the strategy that uses the expiration plugin would end up being expired, but it would be good to either confirm that, or to use different cache names for the CSS and Fonts.

Reply:
K, I can switch to use a different cache name for this example, but it'd probably be good to verify the behavior and, if it's as you fear, add a warning to the lib and/or the docs.

Reply:
Updated.